### DISCUSSION TITLE
how to create my server

### DISCUSSION THREAD

**Discussion by blindrain** [MineLittlePony/MineLittlePony#265] [Q&A] [answered]
how do I setup my dedicated linux Minecraft server to run this ?

**Comment by MimzyFang** [comment, marked as answer]
This mod is completely Clientside you don't need a server to use this

Besides that if you want to run a server I can recommend you to go to the fabric webpage and on there select download for server it will give you all the commands you need to setup a server 
https://fabricmc.net/use/server/
